.eslintignore

```diff
@@ -1 +1,2 @@
-.gitignore
\ No newline at end of file
+.gitignore
+ifrau-client.js
```

Comment:
`eslint` doesn't like dynamic imports in CJS modules, but because it throws a parsing error, exceptions don't work. Hence my solution here is to isolate the dynamic import as much as possible.